config/manager/controller_manager_config.yaml

```diff
@@ -9,8 +9,9 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: 67a0541b.pipeline-service.io
-namespace: admin
+namespace: settings-ps-controller
```

Comment:
I'd rather have full names rather than acronyms.

Reply:
Updated all references of 'settings-ps-controller' to 'settings-pipeline-service-controller'